tests/helpers.py

```diff
@@ -430,3 +434,87 @@ def delete_all_cephfilesystem():
     for item in cephfs_dict:
         assert CFS.delete(resource_name=item.get('metadata').get('name'))
     return True
+
+
+@retry(CephHealthException, tries=5, delay=3, backoff=1)
```

Comment:
I see that we are trying to check ceph health already on multiple places like was done in [deployment](https://github.com/red-hat-storage/ocs-ci/blob/1e5804cd2188c15b6c70045d2473181d058c8dda/tests/conftest.py) would be nice to follow only one code function which can then be extended for more needs if possible. I see that @shylesh is working also on this PR [cluster_health_check](https://github.com/red-hat-storage/ocs-ci/pull/176/files#diff-15c5222219a26f77b1ddd5aeb9b722d8R155). So would be nice to converge on one method.

Reply:
@petr-balogh Done